cg/meta/orders/status.py

```diff
@@ -391,28 +367,37 @@ def store_microbial_order(
                     )
                     self.status.add_commit(organism)
 
-                new_sample = self.status.add_microbial_sample(
+                if comment:
+                    sample_comment = f"Order comment: {comment}"
```

Comment:
So if I understand we store two informations in one field here as a string. Would it not be preferable to have different fields for this information?

Reply:
sure, how?

Reply:
One columns for order comment and one for sample comment

Reply:
removed the order comment completely